yarn-project/simulator/src/avm/journal/journal.test.ts

```diff
@@ -60,22 +61,20 @@ describe('journal', () => {
       const journalUpdates = journal.flush();
       expect(journalUpdates.newNoteHashes).toEqual([utxo]);
     });
+    it('Should maintain nullifiers', async () => {
+      const utxo = new Fr(1);
+      await journal.writeNullifier(utxo);
 
+      const journalUpdates = journal.flush();
+      expect(journalUpdates.newNullifiers).toEqual([utxo]);
+    });
     it('Should maintain l1 messages', () => {
       const utxo = [new Fr(1)];
       journal.writeL1Message(utxo);
 
       const journalUpdates = journal.flush();
       expect(journalUpdates.newL1Messages).toEqual([utxo]);
     });
-
-    it('Should maintain nullifiers', () => {
-      const utxo = new Fr(1);
-      journal.writeNullifier(utxo);
-
-      const journalUpdates = journal.flush();
-      expect(journalUpdates.newNullifiers).toEqual([utxo]);
-    });
   });
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 64 to 78]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Just reordering nullifier stuff to be right after note hash.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4743?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->